packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -278,6 +280,7 @@ export default class New extends Component {
           nfsOptions,
           srUuid
         ),
+      smb: () => createSrSmb(host.id, name.value, description.value, server.value, username.value, password.value),
```

Comment:
```suggestion
      smb: () => createSrSmb(host.id, name.value, description.value, server.value, username.value, password.value, srUuid),
```

Reply:
If I remember correctly @pdonias advised me not to, because srUuid is optionnal in the API.